src/frontend/js/settings.ts

```diff
@@ -36,3 +36,5 @@ export const PAYMENT_SETTINGS = {
 };
 
 export const MOCK_SERVICE_WORKER_ENABLED = false;
+
+export const DEBUG_UNION_RESOURCES_HOOK = process.env.NODE_ENV !== 'production' && true;
```

Comment:
My previous suggestion could be simplified I think 😅
```suggestion
export const DEBUG_UNION_RESOURCES_HOOK = process.env.NODE_ENV !== 'production';
```

Reply:
I'll change it by `export const DEBUG_UNION_RESOURCES_HOOK = process.env.NODE_ENV !== 'production' && false;`
otherwise we'll have too many message in the browser consol